x/sequencer/keeper/hook_listener.go

```diff
@@ -58,12 +73,13 @@ func (hook rollappHook) FraudSubmitted(ctx sdk.Context, rollappID string, height
 	// unbond all other bonded sequencers
 	sequencers := hook.k.GetSequencersByRollappByStatus(ctx, rollappID, types.Bonded)
```

Comment:
what if there are sequencers which are in unbonding status? Look at my next comment but I think we should simply unbond all of the sequencers regardless of their status (and slash proposer obviously)